build.gradle

```diff
@@ -78,18 +79,19 @@ githubRelease {
     owner = 'gradle'
     repo = 'common-custom-user-data-gradle-plugin'
     targetCommitish = 'main'
-    releaseName = gitHubReleaseName()
-    tagName = gitReleaseTag()
+    releaseName = releaseVersion
+    tagName = releaseVersion.map { "v$it" }
     prerelease = false
     overwrite = false
     generateReleaseNotes = false
     body = releaseNotes
+    dryRun = true
 }
 
 def createReleaseTag = tasks.register('createReleaseTag', CreateGitTag) {
     // Ensure tag is created only after a successful publishing
     mustRunAfter('publishPlugins')
-    tagName = gitReleaseTag()
+    tagName = githubRelease.tagName.map { it.toString() }
```

Comment:
Is this the mapping with toString() needed because githubRelease.tagName is a Property<CharSequence> and CreateGitTag.tagName is a Property<String> ?

Reply:
It's required because `githubRelease.tagName` is a `Property<CharSequence>` and,  without the `toString()` mapping, can't be assigned to `CreateGitTag.tagName` which is a `Property<String>`

Reply:
Yep, that's why I meant. I wonder why they used CharSequence???

Reply:
Oh now, all my `Property<String>` and `Property<CharSequence>` got lost in my comment above due to `<` (markdown).

Reply:
Ahh, gotcha. As for why they used `CharSequence`, not sure. Looking at its implementing classes, none of them outside of `String` pop out to me as super useful for setting something as simple as a tag...seems like the other implementations can be easily converted to a `String` anyways. At a quick glance nothing interesting going on in their plugin code with `CharSequence` either. 🤷